### PR TITLE
General Enhancements and Preps for 1.0

### PR DESCRIPTION
**Pull request by 0xk1f0**
## Basic Requirements

- [ ] I have added Documentation
- [x] I have tested my code
- [x] I have synced my branch with main

## Describe your Pull Request, what does it fix or add?

This should be the last remaining PR before the test run this friday and the first full release.

## Is there anything you want to mention? (unchecked code, possible bugs, etc.)

Ig done

### CODE REVIEW

File: src/components/app/lib/admin/Challenge.svelte [hunk 1]
```diff
@@ -162,36 +209,81 @@
...
-            Challenge Difficulty
-            <Select class="mt-2" items={DIFFICULTIES} bind:value={selectedDiff} />
-        </Label>
+        <Label class="mb-2">Challenge Difficulty</Label>
+        <Select
+            defaultClass="text-neutral-900 dark:text-neutral-100 bg-neutral-100 dark:bg-neutral-900"
+            bind:value={selectedDiff}
+            placeholder=""
+        >
+            <option selected value="">None</option>
```
Comment (Merrixx) [inline src/components/app/lib/admin/Challenge.svelte:248]: Why do we now include None as option ?

**Review by Merrixx — APPROVED** [review]
LGTM 👍🏻

File: src/components/app/lib/admin/Challenge.svelte (hunk 1 above)
Comment (0xk1f0) [reply src/components/app/lib/admin/Challenge.svelte:248]: proper placeholder

**Review by felixslama — APPROVED** [review]
LGTM

**Comment by 0xk1f0** [comment]
Alrighty.

**Pull request merged**